File: src/br/com/globalcode/android/SaleFragment.java
package br.com.globalcode.android;

import android.app.Fragment;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class SaleFragment extends Fragment {

	@Override
	public View onCreateView(LayoutInflater inflater, ViewGroup container,
			Bundle savedInstanceState) {
		
		View v = inflater.inflate(R.layout.sale_fragment_layout, container, false);
		return v;
	}
	
}

File: src/br/com/globalcode/android/SaleFragment.scala
package br.com.globalcode.android

import android.app.Fragment
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

class SaleFragment extends Fragment {

	override def onCreateView(inflater: LayoutInflater, container: ViewGroup,
			savedInstanceState: Bundle) =
		inflater.inflate(R.layout.sale_fragment_layout, container, false)
}
